checkov/terraform/checks/resource/azure/AKSUpgradeChannel.py

```diff
@@ -21,7 +21,7 @@ def __init__(self) -> None:
         )
 
     def get_inspected_key(self) -> str:
-        return "automatic_channel_upgrade"
```

Comment:
These are great, @paddymorgan84! Can you also make this one backwards compatible?